lexicons/com/atproto/repo/listRecords.json

```diff
@@ -12,8 +12,9 @@
           "repo": {"type": "string", "format": "at-identifier", "description": "The handle or DID of the repo."},
           "collection": {"type": "string", "format": "nsid", "description": "The NSID of the record type."},
           "limit": {"type": "integer", "minimum": 1, "maximum": 100, "default": 50, "description": "The number of records to return."},
-          "rkeyStart": {"type": "string", "description": "The lowest sort-ordered rkey to start from (exclusive)"},
-          "rkeyEnd": {"type": "string", "description": "The highest sort-ordered rkey to stop at (exclusive)"},
+          "cursor": {"type": "string"},
+          "rkeyStart": {"type": "string", "description": "DEPRECATED: The lowest sort-ordered rkey to start from (exclusive)"},
+          "rkeyEnd": {"type": "string", "description": "DEPRECATED: The highest sort-ordered rkey to stop at (exclusive)"},
```

Comment:
Did you mean to put deprecated on these?

Reply:
yup sorry didn't get to write a descript cause i hopped on a call. my plan was to stop supporting these entirely here in the next week or so once you switch over to `cursor`